File: src/client/java/vergisst/minecraftmod/weaponthrow/client/handler/RenderRegistry.java
package vergisst.minecraftmod.weaponthrow.client.handler;

import net.fabricmc.fabric.api.client.rendering.v1.EntityRendererRegistry;
import vergisst.minecraftmod.weaponthrow.client.render.WeaponThrowRenderer;
import vergisst.minecraftmod.weaponthrow.handler.EntityRegistry;

public class RenderRegistry {

    public static void registerRenderers() {
        EntityRendererRegistry.register(EntityRegistry.INSTANCE.getWEAPONTHROW(), WeaponThrowRenderer::new);

    }
}

File: src/client/kotlin/vergisst/minecraftmod/weaponthrow/client/handler/RenderRegistry.kt
package vergisst.minecraftmod.weaponthrow.client.handler

import net.fabricmc.fabric.api.client.rendering.v1.EntityRendererRegistry
import vergisst.minecraftmod.weaponthrow.client.render.CommonThrowEntityRenderer
import vergisst.minecraftmod.weaponthrow.handler.EntityRegistry

object RenderRegistry {
    fun registerRenderers() {
        EntityRendererRegistry.register(EntityRegistry.COMMON_THROW_ENTITY, ::CommonThrowEntityRenderer)
    }
}